standard_forms/purchase-order/latest/src/template.html

```diff
@@ -9,57 +9,60 @@
       <th>{{ t(k="report.supplier-item-code", f="Supplier's item code") }}</th>
       <th>{{ t(k="report.item-code",f="Item code") }}</th>
       <th>{{ t(k="report.item-name",f="Item name") }}</th>
+      <th>{{ t(k="label.unit", f="Unit") }}</th>
       <th>
-        {{ t(k="label.requested-number-of-units", f="Requested number of units")
+        {{ t(k="label.requested-packs", f="Requested number of packs")
         }}
       </th>
-      <th>{{ t(k="label.unit-name", f="Unit name") }}</th>
       <th>{{ t(k="report.pack-size", f="Pack size") }}</th>
       <th>
         {{ t(k="label.expected-delivery-date", f="Expected delivery date") }}
       </th>
-      <th>{{ t(k="label.comment", f="Comment") }}</th>
       <th>{{ t(k="label.line-cost", f="Line cost") }}</th>
+      <th>{{ t(k="label.comment", f="Comment") }}</th>
     </tr>
   </thead>
   {% for line in data.purchaseOrder.lines.nodes -%}
   <tr class="body_value">
     <td>{{ line.lineNumber }}</td>
-    <td>{{line.supplierItemCode | default(value="")}}</td>
-    <td>{{line.item.code | default(value="")}}</td>
-    <td>{{line.item.name | default(value="")}}</td>
-    <td>{{line.requestedNumberOfUnits | default(value="")}}</td>
-    <td>{{line.item.unitName | default(value="")}}</td>
-    <td>{{line.requestedPackSize | default(value="")}}</td>
-    <td>{{line.expectedDeliveryDate | default(value="")}}</td>
-    <td>{{line.comment | default(value="")}}</td>
-    <td>{{line.lineCost | default(value="")}}</td>
+    <td>{{ line.supplierItemCode | default(value="") }}</td>
+    <td>{{ line.item.code | default(value="") }}</td>
+    <td>{{ line.item.name | default(value="") }}</td>
+    <td>{{ line.item.unitName | default(value="") }}</td>
+    <td>{{ line.requestedNumberOfPacks | default(value="") }}</td>
+    <td>{{ line.requestedPackSize | default(value="") }}</td>
+    <td>{{ line.expectedDeliveryDate | default(value="") }}</td>
+    <td>{{ line.lineCost | default(value="") }}</td>
+    <td>{{ line.comment | default(value="") }}</td>
   </tr>
   {%- endfor %}
 </table>
 
 <table class="right-aligned-table">
-  <tr>
-    <td>{{t(k="label.total-cost", f="Total cost")}}:</td>
-    <td>{{data.purchaseOrder.orderTotalAfterDiscount | default(value="")}}</td>
+  <tr excel-type="total-row">
+    <td>{{ t(k="heading.total", f="Total")}}:</td>
+    <td>{{ data.purchaseOrder.orderTotalAfterDiscount *
+    data.purchaseOrder.foreignExchangeRate | default(value="") | round(precision=2) }} {{ data.purchaseOrder.currency.code
```

Comment:
Showing the currency code instead of symbol for now since we don't have a way to get that from frontend right now, would be nice to implement later. Would link into [this](https://github.com/msupply-foundation/open-msupply/issues/8838) a bit too~